solution/ui/regulations/eregs-vite/src/utilities/api.js

```diff
@@ -281,9 +281,43 @@ const setCacheItem = async (key, data) => {
 };
 
 // ---------- api calls ---------------
-const getLastUpdatedDates = async (apiUrl, title = "42") => {
-    const result = await httpApiGet(`title/${title}/parts`);
-    return Object.fromEntries(new Map(result.map((obj) => [obj.name, obj.date])));
+const getPartTOC = async (title, part) =>
+    httpApiGet(`title/${title}/part/${part}/version/latest/toc`);
+
```

Comment:
these smaller methods were moved higher in the file to avoid linting error where method was used before it was declared.  It works either way due to Javascript's [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting), but it's nice to please the linter and declare methods before using them.